CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+
+## How to request the change 💡 ?
+
+In case you find a missing feature or you have an idea how to enhance existing functionalities - please let us know by creating a new issue. Before opening a new issue please double-check if it's already reported (but don't worry if occurs to be a duplicate - we will manage). If already there - up-vote ⬆️. We will have a look at it faster!
+
+Please elaborate on "why?" - what's the context, what's the benefit and for whom it is. You will find our guides in the template while opening a feature request ticket. This would help us prioritize and submit meaningful changes.
+
+## How to submit a change 🚩 ?
+
+### GitHub Flow
+
+We are following [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow) to collaborate in this repository. In order to submit the changes please do the following:
+
+1. Open a branch
+
+    Please see branching convention below. You can open a branch on remote as well as a fork.
+
+1. Make changes.
+
+     Make sure your code is passing all the checks, it is well documented and tested so as not to decrease existing test coverage 💪.
+
+1. Create a PR(s)
+
+    Please link an issue and make a good change description. Include development assumptions when necessary. Give all the details for a reviewer to efficiently check your changes.
+
+1. Address review comments
+
+    Please apply changes where necessary. If you disagree with a reviewer - please explain why.
+
+1. Merge your PRs and delete a branch
+
+### Branching convention
+
+In case you are working on a task inside one specific repository, please name your branch `<issue_id>_<short_description>` all lowercase. Multiple words of the description should be divided by an underscore (`_`). E.g. `15_fix_spelling_error` in case you try to solve a spelling mistake mentioned in the issue number `15`.
+
+In case you are working on a task from one repository that affects multiple repositories, please always
+name your branches: `<issue_id>_<issue_repo>_<short description>` all lowercase. Multiple words of the description should be divided by an underscore (`_`). E.g. `15_abc_fix_spelling_error` in case you try to solve a spelling mistake inside `xyz` which closes issue `15` inside `abc`.
+
+### Monorepo and staged.dependencies
+
+Sometimes in order to introduce a change you also need a change from an upstream dependent package. We are using [staged.dependencies](https://github.com/openpharma/staged.dependencies) functionality to simulate a monorepo behavior. The configuration is already in the [staged_dependencies.yaml](staged_dependencies.yaml) file. You just need to name the feature branches identically to take advantage of that. Please read the package manual for more details.
```

Comment:
So this seems a little confusing as if you end up using staged.deps you explicitly do not follow the above branching convention